src/Stratis.Features.Collateral/CheckCollateralFullValidationRule.cs

```diff
@@ -45,6 +49,13 @@ public class CheckCollateralFullValidationRule : FullValidationConsensusRule
 
         public override Task RunAsync(RuleContext context)
         {
+            // If this node is not mining on the side-chain, the rule does not apply.
+            if (!this.fullNode.IsMiningOnSideChain())
```

Comment:
Ouch ugly hack.... you could if you prefer inject the `IPoAMiner` to the rule it should work now, hmm unless its in a different assmebly, not sure, it might get messy as you'd need to shuffle interfaces around.

Reply:
Ah yes, why didn't I see that! I will fix this now 👍